sql-ballerina/Module.md

```diff
@@ -0,0 +1,371 @@
+
+#### Creating tables
+
+This sample creates a table with two columns. One column is of type `int`and the other is of type `varchar`.
+The CREATE statement is executed via the `execute` remote function of the client.
+
+```ballerina
+// Create the ‘Students’ table with the  ‘id’, 'name' and ‘age’ fields.
+var ret = dbClient->execute("CREATE TABLE student(id INT AUTO_INCREMENT, " +
+                         "age INT, name VARCHAR(255), PRIMARY KEY (id))");
+if (ret is sql:ExecutionResult) {
+    io:println("Students table create status in DB: ", ret.affectedRowCount);
+} else {
+    error err = ret;
+    io:println("Students table creation failed: ", err.message());
+}
+```
+
+#### Inserting data
+
+This sample shows three examples of data insertion by executing an INSERT statement using the `execute` remote function 
+of the client.
+
+In the first example, the query parameter values are passed directly into the query statement of the `execute` 
+remote function.
+
+```ballerina
+var ret = dbClient->execute("INSERT INTO student(age, name) " +
+                         "values (23, 'john')");
+if (ret is sql:ExecutionResult) {
+    io:println("Inserted row count to Students table: ", ret.affectedRowCount);
+} else {
+    error err = ret;
+    io:println("Insert to Students table failed: ", err.message());
+}
+```
+
+In the second example, the parameter values, which are in local variables are used to parameterize the SQL query in 
+the `execute` remote function. This type of a parameterized SQL query can be used with any primitive Ballerina type 
+like `string`, `int`, `float`, or `boolean` and in that case, the corresponding SQL type of the parameter is derived 
+from the type of the Ballerina variable that is passed in. 
+
+```ballerina
+string name = "Anne";
+int age = 8;
+
+sql:ParameterizedQuery query = `INSERT INTO student(age, name)
+                                values (${age}, ${name})`;
+var ret = dbClient->execute(query);
+if (ret is sql:ExecutionResult) {
+    io:println("Inserted row count to Students table: ", ret.affectedRowCount);
+} else {
+    error err = ret;
+    io:println("Insert to Students table failed: ", err.message());
+}
+```
+
+In the third example, the parameter values are passed as a `sql:TypedValue` to the `execute` remote function. Use the 
+corresponding subtype of the `sql:TypedValue` such as `sql:Varchar`, `sql:Char`, `sql:Integer`, etc, when you need to 
+provide more details such as the exact SQL type of the parameter.
+
+```ballerina
+sql:VarcharValue name = new ("James");
+sql:IntegerValue age = new (10);
+
+sql:ParameterizedQuery query = `INSERT INTO student(age, name)
+                                values (${age}, ${name})`;
+var ret = dbClient->execute(query);
+f (ret is sql:ExecutionResult) {
+    io:println("Inserted row count to Students table: ", ret.affectedRowCount);
+} else {
+    error err = ret;
+    io:println("Insert to Students table failed: ", err.message());
+}
```

Comment:
IMO, we don't need this code. instead of `var`, mention actual type

Reply:
removed var and introduced to actual types in https://github.com/ballerina-platform/module-ballerina-sql/pull/228/commits/d3df4347c964f2a634909920ca548fefdfc1139c